packages/utils/interactive-hooks/src/useKeyProps.windows.ts

```diff
@@ -26,7 +26,7 @@ export function useKeyCallback(userCallback?: KeyCallback, ...keys: string[]) {
 
 export function getKeyCallbackWorker(userCallback?: KeyCallback, ...keys: string[]) {
   const onKeyEvent = (args: KeyPressEvent) => {
-    if (userCallback !== undefined && (keys === undefined || keys.includes(args.nativeEvent.key))) {
+    if (userCallback !== undefined && !isModifierKey(args.nativeEvent) && (keys === undefined || keys.includes(args.nativeEvent.key))) {
```

Comment:
I think we can extend the "isModifierKey" check to macOS too. We should have the same type of native event (alt / ctrl / meta).

Reply:
I added it to macOS. Could you please check if it works?